docs/03-module-block-3/02-learning-goals.adoc

```diff
@@ -31,20 +31,20 @@ tbd.
   methods.
 
 [[LG-3-2]]
-==== LG 3-2: Relevant Qualities
+==== LG 3-2: Know relevant qualities
```

Comment:
`Know relevant qualities that indicate the use of formal methods`
It's also more precise than just "know relevant qualities" 🙂